test/common/ssl/test_data/certs.sh

```diff
@@ -55,5 +56,9 @@ openssl rand 80 > ticket_key_a
 openssl rand 80 > ticket_key_b
 openssl rand 79 > ticket_key_wrong_len
 
+# Generate unittest_expired_cert.pem (will fail on Mac OS 10.13+, see README.md).
```

Comment:
The `README.md` doesn't really explain why it fails, so it's useless reference... Maybe just say that's because of negative `-days` parameter?